### PR TITLE
chore: updated dependencies

### PR DESCRIPTION
**Pull request by timidak**
- updated dependencies
- added helpers for eslint rules and testing
- fixed lint errors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,39 +17,47 @@
...
-    "lint-fix": "npx eslint . --config=base.js --fix",
-    "test": "eslint ./tests/*.js --config=base.js",
+    "lint": "eslint ./*.js --config=base.js",
+    "lint-fix": "eslint ./*.js --config=base.js --fix",
+    "test": "npm-run-all test:*",
     "test:base": "eslint ./tests/base/*.js --max-warnings 0 --config=base.js",
     "test:typescript": "eslint tests/typescript/*.ts --max-warnings 0 --config=tests/typescript/.eslintrc",
-    "release": "standard-version"
+    "release": "standard-version",
+    "find-unused": "eslint-find-rules -u -n -d ./base.js",
```
Comment (JoeTypeDigital) [inline package.json:35]: is it just on my end, or does "find-unused" already find tons of unused rulesets?

File: package.json [hunk 2]
```diff
@@ -17,39 +17,47 @@
...
+    "find-unused:ts": "eslint-find-rules -u -n -d ./tests/typescript/.eslintrc"
```
Comment (JoeTypeDigital) [inline package.json:36]: same here.
What are the criteria for a rule being unused? I'm asking cause e.g. "no-precision-loss" is set to "warn" but it's being labeled "unused".

**Review by JoeTypeDigital — APPROVED** [review]
1 minor q but that shouldn't stop this pr. The unused tools seem like a great idea. The rest looks fine 💯

File: package.json (hunk 2 above)
Comment (timidak) [reply package.json:36]: We should investigate that would you open an investigation ticket? @JoeTypeDigital

File: package.json (hunk 1 above)
Comment (timidak) [reply package.json:35]: It does find a ton of unused rules, yes. So we should tackle them in the next version I guess. What do you mean? @JoeTypeDigital

**Pull request merged**